pom.xml

```diff
@@ -51,7 +51,7 @@
         <connection>scm:git:https://gitbox.apache.org/repos/asf/struts.git</connection>
         <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/struts.git</developerConnection>
         <url>https://github.com/apache/struts/</url>
-        <tag>STRUTS_2_5_16</tag>
+        <tag>STRUTS_2_5_19</tag>
```

Comment:
Hello @lukaszlenart . 
Wasn't absolutely sure about that one either, but since it said 2_5_16 for the 2.5.18 build the guess was it should match the level of the current build.  If it should be current build-1, that can be fixed ...

Reply:
This is a pure Git thing and purely informative, I think it can be also `support-2-3` to point the branch.